fitbenchmarking/fitbenchmarking
# Update to Mantid v6.2 for our tests

#### Description of Work
This PR updates the version of Mantid we use in our tests to the latest version of Mantid (6.2).

Fixes #940


#### Testing Instructions

1. Make sure the tests are passing

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
